### ISSUE TITLE
Feature Request :queue and task progress

### ISSUE THREAD

**Issue by dduuyt** [yuudi/rclone-webui-angular#20]
Can queue tasks and show Transferring progress on dashboard

**Comment by yuudi** [comment]
It is an important feature that I have been wanting to add, but the backend does not provide API for this.

Now the only thing frontend can do is to display the job that is currently processing, you can view it on job tab.

There are several limitations on backend API:
- job process percentage is not shown
- finished job cannot be listed
- job-id may not be relayed on because backend may restart

So before the task progress can be comfortably viewed on frontend, there are still  something to do at backend. I have been making pull request on rclone backend to improve these, but it still need some time.

**Comment by sjtuross** [comment]
I find this PR https://github.com/yuudi/rclone-webui-angular/pull/7 was merged, but I can't find the job tab in webui. Did I miss anything?

**Comment by yuudi** [comment]
> I find this PR #7 was merged, but I can't find the job tab in webui. Did I miss anything?

Yes it is completed but it is not satisfying, so I hide the entry of the jobs tab. If you want you can go directly to `.../rclone/job` to see it.

The rclone backend api of job status is not stable, and we are not able to query completed job from backend, so some behavior of the job manager might be confusing and misleading.

**Comment by sjtuross** [comment]
Thank you for your reply. Besides completed jobs, I'm actually more interested in monitoring running jobs with details like filename, speed etc. Currently it seems this hidden page `rclone/job` doesn't show running jobs.
